src/graphql/schema.js

```diff
@@ -188,6 +188,18 @@ type PaginatedTopics {
   pageInfo: PageInfo
 }
 
+
+type AccumulatedVotes {
+  voterAddress: String!
+  amount: String!
+}
+
+type PaginatedVotes {
```

Comment:
change to `PaginatedAccumulatedVotes`. we should be consistent and have `PaginatedVotes` on the Vote table